LinuxPackaging.cmake

```diff
@@ -0,0 +1,95 @@
+# support creating some basic binpkgs via `make package`
+set(CPACK_GENERATOR TGZ;RPM;DEB)
+
+set(CPACK_INCLUDE_TOPLEVEL_DIRECTORY 0)
+set(CPACK_PACKAGE_VERSION_MAJOR ${LIBIIO_VERSION_MAJOR})
+set(CPACK_PACKAGE_VERSION_MINOR ${LIBIIO_VERSION_MINOR})
+set(CPACK_PACKAGE_VERSION_PATCH g${LIBIIO_VERSION_GIT})
+set(CPACK_BUNDLE_NAME libiio)
+set(CPACK_PACKAGE_VERSION ${LIBIIO_VERSION})
+# debian specific package settings
+set(CPACK_PACKAGE_CONTACT
+       "Engineerzone <https://ez.analog.com/community/linux-device-drivers>")
+
+FIND_PROGRAM(DPKG_CMD dpkg)
+FIND_PROGRAM(DPKGQ_CMD dpkg-query)
```

Comment:
Wait, you're right, it shouldn't exit the sub-script.
My point was that it shouldn't look for dpkg and dpkg-query if that option is not enabled.